<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWindow</class>
 <widget class="QMainWindow" name="FilterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="3" colspan="5">
     <widget class="QTabWidget" name="tab_table">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_education">
       <attribute name="title">
        <string>Стажування</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QTableView" name="lv_education"/>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>330</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Setting search</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="11" column="0" colspan="2">
            <widget class="QComboBox" name="cb_education_e"/>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="le_title_e"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="le_description_e"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="le_start_e"/>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="le_end_e"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="le_req_exp_e"/>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="le_option_condition_e"/>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLineEdit" name="le_city_e"/>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="le_location_e"/>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLineEdit" name="le_phone_e"/>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QLineEdit" name="le_email_e"/>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QLineEdit" name="le_company_e"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_resume">
       <attribute name="title">
        <string>Резюме</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>330</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Параметри пошуку</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="9" column="0" colspan="2">
            <widget class="QLineEdit" name="le_a_r">
             <property name="placeholderText">
              <string>Про себе</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLineEdit" name="le_email_r">
             <property name="placeholderText">
              <string>Е-пошта</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="le_hob_r">
             <property name="placeholderText">
              <string>Хоббі</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="le_pip_r">
             <property name="placeholderText">
              <string>Піп</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="le_l_r">
             <property name="placeholderText">
              <string>Знання мов</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="le_s_r">
             <property name="placeholderText">
              <string>Комунікативні уміння</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="le_ph_r">
             <property name="placeholderText">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="le_h_r">
             <property name="placeholderText">
              <string>Професійні уміння</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="le_stype_r">
             <property name="placeholderText">
              <string>Skype</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLineEdit" name="le_q_r">
             <property name="placeholderText">
              <string>Клаліфікація</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableView" name="lv_resume"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_vacancy">
       <attribute name="title">
        <string>Вакансії</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>330</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Параметри пошуку</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="le_city_v">
             <property name="placeholderText">
              <string>Місто</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLineEdit" name="le_company_v">
             <property name="placeholderText">
              <string>Компанія</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="le_title_v">
             <property name="placeholderText">
              <string>Заголовок</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="le_exp_v">
             <property name="placeholderText">
              <string>Досвід</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="le_description_v">
             <property name="placeholderText">
              <string>Опис</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="le_email_v">
             <property name="placeholderText">
              <string>Е-пошта</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="le_location_v">
             <property name="placeholderText">
              <string>Локація</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="le_salary_v">
             <property name="placeholderText">
              <string>Зарплата</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLineEdit" name="le_phone_v">
             <property name="placeholderText">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableView" name="lv_vacancy"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn_search">
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
